docs/group_faqs/gf_faqs.rst

```diff
@@ -7,3 +7,29 @@ A: On the Solace Cloud `Create Service <https://console.solace.cloud/services/cr
 
 .. image:: ../img/faqs_enterprise_access_1.png
    :target: https://console.solace.cloud/services/create
+
+Q: What clouds and regions are available?
+
+A: This information is available at the bottom of our `pricing page <https://cloud.solace.com/pricing/>`_.
+
+Q: Why aren't all of the regions listed on the `pricing page <https://cloud.solace.com/pricing/>`_ available when I'm creating a service?
+
+A: Only certain regions are available when creating a Free service.  Free service regions are marked with an asterisk on our `pricing page <https://cloud.solace.com/pricing/>`_.  Even if you're not using a Free service, you may notice that some regions are not available.  This is because we list all of the "Available" regions on our website, while only the "Enabled" regions can be used when creating a service.
+
+Q: What is the difference between an "Available" region and an "Enabled" region?
+
+A: "Available" regions are those which have been identified as having the IaaS that is required to run our messaging services.  "Enabled" regions are those in which PubSub+ Cloud infrastructure is installed and ready to create messaging services.
```

Comment:
```suggestion
A: "Available" regions are those which have been identified as having the IaaS resources that is required to run our messaging services.  "Enabled" regions are those in which PubSub+ Cloud infrastructure is installed and ready to create messaging services.
```

Reply:
should be "IaaS resources that are" so I'll change